packages/babel-parser/test/fixtures/comments/attachComment-false/array-expression-trailing-comma/output.json

```diff
@@ -0,0 +1,244 @@
+        "declarations": [
+          {
+            "type": "VariableDeclarator",
+            "start":84,"end":162,"loc":{"start":{"line":8,"column":6},"end":{"line":13,"column":1}},
+            "id": {
+              "type": "Identifier",
+              "start":84,"end":102,"loc":{"start":{"line":8,"column":6},"end":{"line":8,"column":24},"identifierName":"trailingAfterComma"},
+              "name": "trailingAfterComma"
+            },
+            "init": {
+              "type": "ArrayExpression",
+              "start":105,"end":162,"loc":{"start":{"line":8,"column":27},"end":{"line":13,"column":1}},
+              "extra": {
+                "trailingComma": 140
+              },
+              "elements": [
+                {
+                  "type": "StringLiteral",
+                  "start":109,"end":114,"loc":{"start":{"line":9,"column":2},"end":{"line":9,"column":7}},
+                  "extra": {
+                    "rawValue": "One",
+                    "raw": "\"One\""
+                  },
+                  "value": "One"
+                },
+                {
+                  "type": "StringLiteral",
+                  "start":135,"end":140,"loc":{"start":{"line":11,"column":2},"end":{"line":11,"column":7}},
+                  "extra": {
+                    "rawValue": "Two",
+                    "raw": "\"Two\""
+                  },
+                  "value": "Two"
+                }
+              ]
+            }
+          }
+        ],
+        "kind": "const"
+      },
+      {
+        "type": "VariableDeclaration",
+        "start":164,"end":220,"loc":{"start":{"line":15,"column":0},"end":{"line":18,"column":1}},
+        "declarations": [
+          {
+            "type": "VariableDeclarator",
+            "start":170,"end":220,"loc":{"start":{"line":15,"column":6},"end":{"line":18,"column":1}},
+            "id": {
+              "type": "Identifier",
+              "start":170,"end":188,"loc":{"start":{"line":15,"column":6},"end":{"line":15,"column":24},"identifierName":"trailingAfterArray"},
+              "name": "trailingAfterArray"
+            },
+            "init": {
+              "type": "ArrayExpression",
+              "start":191,"end":220,"loc":{"start":{"line":15,"column":27},"end":{"line":18,"column":1}},
+              "extra": {
+                "trailingComma": 200
+              },
+              "elements": [
+                {
+                  "type": "StringLiteral",
+                  "start":195,"end":200,"loc":{"start":{"line":16,"column":2},"end":{"line":16,"column":7}},
+                  "extra": {
+                    "rawValue": "One",
+                    "raw": "\"One\""
+                  },
+                  "value": "One"
+                }
+              ]
+            }
+          }
+        ],
+        "kind": "const"
+      }
+    ],
+    "directives": [],
+    "comments": [
```

Comment:
is it expected to have two sets of comments arrays? Though not sure why you're change would do that, or it was like this before?